### PR TITLE
[Sepolia] upgrade sepolia system config to max gas limit 400m

### PR DESCRIPTION
**Pull request by 0x00101010**
Adds scripts for the following:

Deploy new L1 systemconfig that has the following 2 changes:
a. The max_gas_limit is 400_000_000, instead of current value of 200_000_000
b. The version is set to 2.2.0+max-gas-limit-400M"
Upgrade the existing systemconfig to the new implementation contract

### CODE REVIEW

**Comment by cb-heimdall** [comment]
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>

File: sepolia/2024-12-06-increase-max-gas-limit/.env [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+OPTIMISM_REPO=https://github.com/ethereum-optimism/optimism.git
+OPTIMISM_VERSION=op-contracts/v1.5.0
+OPTIMISM_CONTRACT_PATCH=patch/max-gas-limit.patch
+
+SAFE_ADDRESS=0x0fe884546476dDd290eC46318785046ef68a0BA9
+PROXY_ADMIN_ADDRESS=0x0389E59Aa0a41E4A413Ae70f0008e76CAA34b1F3
+SYSTEM_CONFIG_ADDRESS=0xf272670eb55e895584501d564AfEB048bEd26194
+
+NEW_IMPLEMENTATION=0x31aad1062c25ce545c14bd9ee64dedef6c6b6fac
```
Comment (0x00101010) [inline sepolia/2024-12-06-increase-max-gas-limit/.env:11]: https://sepolia.etherscan.io/address/0x31aad1062c25ce545c14bd9ee64dedef6c6b6fac#code

**Review by angel-ding-cb — DISMISSED** [review]

**Review by meyer9 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for meyer9 @ 2024-12-06 18:13:30 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by meyer9 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for meyer9 @ 2024-12-06 18:13:53 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by meyer9 — APPROVED** [review]

**Comment by cb-heimdall** [comment]
Review Error for meyer9 @ 2024-12-06 18:15:02 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by mdehoog — APPROVED** [review]

**Pull request merged**